yellowbrick/text/postag.py

```diff
@@ -139,16 +144,34 @@ def fit(self, X, y=None, **kwargs):
         self : instance
             Returns the instance of the transformer/visualizer
         """
-        # TODO: add support for other tagsets?
-        if self.tagset == "penn_treebank":
-            self.pos_tag_counts_ = self._penn_tag_map()
-            self._handle_treebank(X)
-
-        elif self.tagset == "universal":
-            self.pos_tag_counts_ = self._uni_tag_map()
-            self._handle_universal(X)
+        if(self.stack == True and y == None):
```

Comment:
Normally we wouldn't be so nitpicky, but we're looking forward to working with you on more PRs and so I wanted to mention some [style guide](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) recommendations that we use in Yellowbrick to make our code perform better and protect it from future changes. In this case, we would probably write:

```python
if self.stack and y is None:
```

Because `self.stack` is a bool and the `is` comparator makes sense a bit more sense for `None`.  Note that in some cases we may also do `if self.stack and y` - which only checks for the truthiness of y, particularly failing if `y` is `[]` -- however, since it is not necessarily appropriate to do a `len(X) == len(y)` check here (there are other tools that make sure `X` and `y` are correctly aligned), the `y is None` check is probably best.